### DISCUSSION TITLE
2017 vs. 2022 NAICS Codes in v1.2 and Future Releases

### DISCUSSION THREAD

**Discussion by ghost** [USEPA/supply-chain-factors#66] [Q&A]
V1.2 of the model uses 2017 NAICS codes, but 2020 NAICS codes are the most recent. Do you anticipate when the model will be revised and mapped to the 2020 NAICS codes?

**Comment by bl-young** [comment]
There are no immediate plans to provide the data using the 2022 NAICS codes. In future releases we may update the target NAICS year. The concordances provided by US Census may be helpful if you want to convert from 2017 to 2022: https://www.census.gov/naics/?68967

**Comment by WesIngwersen** [comment]
I corrected this topic name to reflect 2022 codes. Yes, the 2022 are the most recent. Future versions will likely move to this, but we haven't decided yet when.
